extensions/amp-stream-gallery/validator-amp-stream-gallery.protoascii

```diff
@@ -5,6 +5,7 @@ tags: {  # amp-stream-gallery
     name: "amp-stream-gallery"
     version: "1.0"
     version: "latest"
+    bento_supported_version: "1.0"
```

Comment:
This one includes `latest`, since `1.0` is the only valid version.

Reply:
Done in #35762